test/specs/e2e_wiki.js

```diff
@@ -0,0 +1,52 @@
+
+import MainPage from '../pageobjects/main.page.js'
+import mainPage from "../pageobjects/main.page.js";
```

Comment:
I guess those 2 import statements `MainPage` and `mainPage` are the same one, right? You can keep the `MainPage` and then remove the `mainPage` import statement since this is the same page-object.